android/start.md

```diff
@@ -100,19 +109,30 @@ b. To generate the Java classes for these models, click the Gradle Task dropdown
 
 ## Step 3: Add API and Database
 
-a. Run `amplify configure` from the root of your application folder to set up Amplify with your AWS account.
+a. Run `amplify configure` in Terminal from the root of your application folder to set up Amplify with your AWS account.
+
+    - Browser will pop up prompting you to sign in / create new account
```

Comment:
Your default browser will open a tab prompting you to sign in / create a new AWS account